### DISCUSSION TITLE
Creating two one-to-many relationship between two tables

### DISCUSSION THREAD

**Discussion by shamsch** [prisma/prisma#14469] [Q&A] [answered]
I need to make this one-to-many relationship between two tables, but there is two relations between the table referring to the same field. So this is my model. As you can see, I have both `departure_station_id` and `return_station_id` in `Journey` model. How can I now create a relationship between those fields with the `Station` model referring to the `station_id` field?
 
```prisma
model Station {
  id       Int    @id @default(autoincrement())
  name     String
  address  String
  city     String
  operator String
  capacity Int
  lon      Float
  lat      Float
}

model Journey {
  departure_time       DateTime
  return_time          DateTime
  departure_station    String
  departure_station_id Int
  return_station       String
  return_station_id    Int
  duration             Float
  covered_distance     Float
  month                String
  id                   Int      @id @default(autoincrement())
}
```

**Comment by shamsch** [comment, marked as answer]
Hey, I found the answer in the documentation. Please check [this](https://www.prisma.io/docs/concepts/components/prisma-schema/relations#disambiguating-relations).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
